processor/groupbyattrsprocessor/processor_test.go

```diff
@@ -139,6 +142,62 @@ func someComplexMetrics(withResourceAttrIndex bool, rmCount int, ilmCount int, d
 	return metrics
 }
 
+func someComplexHistogramMetrics(withResourceAttrIndex bool, rmCount int, ilmCount int, dataPointCount int, histogramSize int) pdata.Metrics {
```

Comment:
I believe the return type should be `pmetric.Metric`

Reply:
Fixed